egs/librispeech/ASR/distillation_with_hubert.sh

```diff
@@ -0,0 +1,95 @@
+stage=$1
+export CUDA_VISIBLE_DEVICES="2,3,4,5"
+
+
+if [ $stage -eq 0 ]; then
```

Comment:
Should it be `-leq 0`? Otherwise, users have to invoke this script for multiple times.

Reply:
Actually I specially use -eq instead of -leq or -le.
Because distillation with vq is quite a new concept in icefall.
 I hope users could run this example script stage by stage, before they know what each stage really does.